docs/specs/meta-clients/web3wallet/sdk-api.md

```diff
@@ -71,7 +71,7 @@ class Web3Wallet {
   public abstract getPendingAuthRequests(): Promise<Record<number, PendingRequest>>;
   
   // register device token for Push server (BOTH)
-  public abstract registerDeviceToken(token: string): Promise<void>;
+  public abstract registerDeviceToken(token: string, enableAlwaysDecrypted?: boolean = false): Promise<void>;
```

Comment:
enableAlwaysDecrypted - > maybe a different name?
@llbartekll

Reply:
i would call them encrypted as they are e2ee.

Reply:
also im not sure "always" adds any value in this naming 🤔

Reply:
Yeah "always" is redundant but still not sure about encrypted vs decrypted. have to sleep on it :)

Reply:
enableEncrypted? wdyt @jakubuid

Reply:
How about "enableDetailed" or "enableAdvanced"?

Reply:
`enableEncrypted` let it be